pkg/deployer/wait/wait.go

```diff
@@ -98,15 +98,9 @@ func (wt *Waiter) Interval(iv time.Duration) *Waiter {
 	return wt
 }
 
-func (wt Waiter) ForNamespaceDeleted(ctx context.Context, namespace string) error {
-	log := wt.Log.WithValues("namespace", namespace)
-	log.Info("wait for the namespace to be gone")
-	return k8swait.PollImmediate(wt.PollInterval, wt.PollTimeout, func() (bool, error) {
-		nsKey := ObjectKey{Name: namespace}
-		ns := corev1.Namespace{} // unused
-		err := wt.Cli.Get(ctx, nsKey.AsKey(), &ns)
-		return deletionStatusFromError(wt.Log, "Namespace", nsKey, err)
-	})
+func (wt *Waiter) Steps(st int) *Waiter {
+	wt.PollSteps = st
+	return wt
```

Comment:
where is this used?

Reply:
You are right, this one escape to my deletion effort.

Thanks